[2023-06-10T04:21:25.838+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: fourth_dag.fourth_task scheduled__2023-05-14T00:00:00+00:00 [queued]>
[2023-06-10T04:21:26.190+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: fourth_dag.fourth_task scheduled__2023-05-14T00:00:00+00:00 [queued]>
[2023-06-10T04:21:26.197+0000] {taskinstance.py:1308} INFO - Starting attempt 1 of 6
[2023-06-10T04:21:26.537+0000] {taskinstance.py:1327} INFO - Executing <Task(BashOperator): fourth_task> on 2023-05-14 00:00:00+00:00
[2023-06-10T04:21:26.844+0000] {standard_task_runner.py:57} INFO - Started process 5311 to run task
[2023-06-10T04:21:27.031+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'fourth_dag', 'fourth_task', 'scheduled__2023-05-14T00:00:00+00:00', '--job-id', '492', '--raw', '--subdir', 'DAGS_FOLDER/second_dag.py', '--cfg-path', '/tmp/tmp_h2azf_r']
[2023-06-10T04:21:27.065+0000] {standard_task_runner.py:85} INFO - Job 492: Subtask fourth_task
[2023-06-10T04:21:28.087+0000] {task_command.py:410} INFO - Running <TaskInstance: fourth_dag.fourth_task scheduled__2023-05-14T00:00:00+00:00 [running]> on host 93a4f35889be
[2023-06-10T04:21:29.694+0000] {taskinstance.py:1547} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='madhu' AIRFLOW_CTX_DAG_ID='fourth_dag' AIRFLOW_CTX_TASK_ID='fourth_task' AIRFLOW_CTX_EXECUTION_DATE='2023-05-14T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-05-14T00:00:00+00:00'
[2023-06-10T04:21:29.710+0000] {subprocess.py:63} INFO - Tmp dir root location: 
 /tmp
[2023-06-10T04:21:29.714+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'echo "hello world i am madhu this is fourth task"']
[2023-06-10T04:21:29.806+0000] {subprocess.py:86} INFO - Output:
[2023-06-10T04:21:29.828+0000] {subprocess.py:93} INFO - hello world i am madhu this is fourth task
[2023-06-10T04:21:29.829+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2023-06-10T04:21:30.026+0000] {taskinstance.py:1350} INFO - Marking task as SUCCESS. dag_id=fourth_dag, task_id=fourth_task, execution_date=20230514T000000, start_date=20230610T042125, end_date=20230610T042130
[2023-06-10T04:21:30.706+0000] {local_task_job_runner.py:225} INFO - Task exited with return code 0
[2023-06-10T04:21:30.913+0000] {taskinstance.py:2651} INFO - 0 downstream tasks scheduled from follow-on schedule check
